app/src/main/java/org/kiwix/kiwixmobile/main/KiwixMainActivity.kt

```diff
@@ -79,4 +119,25 @@ class KiwixMainActivity : CoreMainActivity() {
       it.onNewIntent(intent, this)
     }
   }
+
+  override fun setupDrawerToggle(toolbar: Toolbar) {
+    drawerToggle =
+      ActionBarDrawerToggle(
+        this, new_navigation_container, toolbar, R.string.open, R.string.close_all_tabs
+      )
+    new_navigation_container.addDrawerListener(drawerToggle)
+    drawerToggle.syncState()
+  }
+
+  override fun onNavigationItemSelected(item: MenuItem): Boolean {
+    when (item.itemId) {
+      R.id.menu_host_books -> start<ZimHostActivity>()
+      else -> return super.onNavigationItemSelected(item)
+    }
+    return true
```

Comment:
this statement is unreachable I would think? Ah no because `host_books` doesn't return a boolean the `else` return doesn't consume all pathways, eh this is fine